app/javascript/styles/mastodon/polls.scss

```diff
@@ -121,15 +121,10 @@
       border-radius: 4px;
     }
 
-    &.active {
-      border-color: $valid-value-color;
-      background: $valid-value-color;
-    }
```

Comment:
I missed it 😩

Reply:
added .active back here https://github.com/mastodon/mastodon/pull/26162

Just worried it's too dark for dark theme.

Reply:
Note: had this not been caught, and not been fixed, this would've resulted in Moderator having zero idea why reports had been submitted, as the styling for Polls is reused by the `ReportReasonSelector` 

It may be a very good idea to untangle these two, such that changes to the polls style can't accidentally break the moderation UI.

<img width="1418" alt="Screenshot 2023-07-27 at 4 35 39 am" src="https://github.com/mastodon/mastodon/assets/30827/e599ea51-f95b-4c5c-b7be-5dfa7dfee426">